src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/writetag/InputWriteTagIgnore.java

```diff
@@ -1,7 +1,7 @@
 /*
 WriteTag
 tag = @tag2
-tagFormat = \S
```

Comment:
In Test method it was
>checkConfig.addProperty("tag", "@todo2");

Please recheck all input files to preserve all settings as is.